src/CaptureFile/CaptureSectionInputStreamImpl.h

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef CAPTURE_SECTION_INPUT_STREAM_IMPL_H_
+#define CAPTURE_SECTION_INPUT_STREAM_IMPL_H_
+
+#include "CaptureFile/CaptrureSectionInputStream.h"
+#include "FileFragmentInputStream.h"
+#include "OrbitBase/File.h"
+
+namespace orbit_capture_file_internal {
+
+class CaptureSectionInputStreamImpl : public orbit_capture_file::CaptureSectionInputStream {
```

Comment:
Could you add a brief comment to the class?

Reply:
done